tests/mc_simulator/test_mc_simulator.py

```diff
@@ -816,9 +792,9 @@ def test_MantarrayMcSimulator__processes_testing_commands_during_reboot(
     )
 
     # send reboot command
-    dummy_timestamp = 0
+    expected_timestamp = 1000
```

Comment:
it appears these numbers may just be arbitrary to inject some variety into the test suite. If they're truly arbitrary, a good way to indicate that (and inject even more variety) is to actually have it be a randomized number (within some range).  `pytest-randomly` lets us deterministically re-run a test suite with the same randomly generated numbers, since it controls the random seed, so that avoids any concern about not being able to debug a failing test